CloudKitty/api/Dockerfile

```diff
@@ -7,7 +7,12 @@ RUN  apt-get update \
         gcc \
         python3-dev \
     && pip install cloudkitty \
-    && mkdir /etc/cloudkitty \
+    && mkdir \
+        /etc/cloudkitty \
+        /etc/cloudkitty-config \
```

Comment:
Why not simply `/config` ? would be simpler to use

Reply:
I'm not a huge fan of just /config, i don't want to break too many habits when switching from ck −> ck on docker.

Reply:
You don't switch from pure ck to ck on docker, you switch from one automated deployment method (OSA, kolla-ansible, anything). You rarely directly edit `/etc/cloudkitty/config_x`. And even if you want to edit stuff in `/etc/cloudkitty`, there are symbolic links now